src/modules/locations/_mocks__/locations.service.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  locationTestStub1,
+  locationsByStateStub,
+  locationsStub
+} from "../stubs/locations-service.stub";
+
+export const LocationsServiceMock = jest.fn().mockReturnValue({
```

Comment:
Para o `jest.mock` identificar automaticamente o mock dessa service, ela tem q ter o mesmo nome da service original: `LocationsService`

Reply:
O problema é que no beforeAll, pelo que entendi, o createTestingModule faz a injeção do mock no lugar do serviço. Só que da forma como está tem que invocar LocationsServices também. A ter que gerar um alias eu prefiro mudar o nome da mock.

o jest.mock tá comentado (tirei nessa revisão). Talvez se for injetar o jest.mock o beforeAll não seja mais necessário. Mas ai precisaria testar/ler a doc um pouco mais. 
![image](https://github.com/otaldonuness/meupetminhavida-back/assets/9803589/720a2c0c-fd0f-4414-9b44-3da1b97aef2c)